apps-rendering/src/components/Layout/LiveLayout.tsx

```diff
@@ -114,9 +115,10 @@ const keyEvents = (blocks: LiveBlock[]): KeyEvent[] =>
 
 interface Props {
 	item: LiveBlog | DeadBlog;
+	edition: Edition;
```

Comment:
Should `edition` live on the `Item` object? `Item` is a description of the article to be rendered, and I think the edition could be considered part of that description?